src/components/xy_chart/axis/x_axis.js

```diff
@@ -0,0 +1,67 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { XAxis } from 'react-vis';
+import { EuiXYChartAxisUtils } from '../utils/axis_utils';
+
+const { TITLE_POSITION, ORIENTATION } = EuiXYChartAxisUtils;
+
+export class EuiXAxis extends PureComponent {
+  render() {
+    const {
+      title,
+      titlePosition,
+      orientation,
+      tickSize,
+      tickLabelAngle,
+      tickFormat,
+      tickValues,
+      onZero,
+      ...rest
+    } = this.props;
+    return (
+      <XAxis
+        title={title}
+        position={titlePosition}
+        orientation={orientation}
+        on0={onZero}
```

Comment:
Shouldn't this be `onZero` instead of `on0`?
Btw. `onZero` sounds more like an event handler than a boolean. Can't come up with a better name from the top of my head though.

Reply:
`on0` is how react-vis call it. We changed to `onZero` on our components.  Maybe something like `isOnZero` sounds more like a bool

Reply:
agreed, `isOnZero` is a better name